driver-sync/src/main/com/mongodb/client/internal/ClientEncryptionImpl.java

```diff
@@ -37,7 +37,7 @@
 
     public ClientEncryptionImpl(final ClientEncryptionSettings options) {
         this.keyVaultClient = MongoClients.create(options.getKeyVaultMongoClientSettings());
-        this.crypt = Crypts.create(SimpleMongoClients.create(keyVaultClient), options);
+        this.crypt = Crypts.create(keyVaultClient, options);
```

Comment:
No longer need SimpleMongoClient to unify things, since we can just use the delegate directly now.

Reply:
:+1: